TrackParamFilter/sections/specific.txt

```diff
@@ -1,6 +1,9 @@
 !
-! The rules for specific sites
+! This section contains the list of domain-specific rules that remove URL tracking parameters. Rules have to contain only a single modifier: `removeparam`.
```

Comment:
" only a single modifier: `removeparam`."
content type and `domain`  are also modifiers)